script.js

```diff
@@ -1,51 +1,63 @@
-  count = userData.length;
-} else {
-  userData = [];
-  localStorage.setItem('formdata', JSON.stringify(userData));
-}
+// const Book = require("book.js");
 
-/* eslint no-unused-vars: "off" */ /* eslint eqeqeq: "off" */
-function deleteItem(id) {
-  // console.log(id);
-  const filteredArray = userData.filter((e) => e.id != id);
-  localStorage.setItem('formdata', JSON.stringify(filteredArray));
-}
+class BookManager {
+  constructor() {
+    this.button = document.querySelector('button');
+    this.title = document.querySelector('.title');
+    this.author = document.querySelector('.author');
+    this.content = document.querySelector('.books');
+    this.userData = JSON.parse(localStorage.getItem('formdata')) || [];
+    this.count = this.userData.length;
+    this.fetchBooks();
+    this.setupEventListeners();
+  }
 
-function fetchBooks() {
-  let placeholder = '';
-  userData.forEach((book) => {
-    placeholder += `
-                <div>
-                    <h3>${book.title}</h3>
-                    <p>${book.author}</p>
-                </div>
-                <button onclick="deleteItem('${book.id}')">Remove</button>
-                <hr>
-            `;
-  });
-  content.innerHTML = placeholder;
+  setupEventListeners() {
+    this.button.addEventListener('click', (e) => {
+      e.preventDefault();
+      const formData = new Book(this.count + 1, this.title.value, this.author.value);
+      this.userData.push(formData);
+      localStorage.setItem('formdata', JSON.stringify(this.userData));
+      this.count += 1;
+      this.renderBooks();
+    });
 
-  button.addEventListener('click', (e) => {
-    e.preventDefault();
-    const formData = {
-      id: count += 1,
-      title: title.value,
-      author: author.value,
-    };
-    // console.log(userData);
-    userData.push(formData);
-    // console.log(userData);
-    localStorage.setItem('formdata', JSON.stringify(userData));
-  });
-}
+    this.content.addEventListener('click', (e) => {
+      if (e.target.classList.contains('remove-btn')) {
+        const id = parseInt(e.target.dataset.id, 10);
+        this.deleteItem(id);
+      }
+    });
+  }
 
-fetchBooks();
+  deleteItem(id) {
+    this.userData = this.userData.filter((book) => book.id !== id);
+    localStorage.setItem('formdata', JSON.stringify(this.userData));
+    this.renderBooks();
+  }
+
+  renderBooks() {
+    let placeholder = '';
+    this.userData.forEach((book, index) => {
+      const backgroundColor = index % 2 === 0 ? 'white' : '#d2d2d2';
+      placeholder += `
+       <div class="book-item" style="background-color: ${backgroundColor};">
+        <div class="book" >
+        <h3>"${book.title}" by ${book.author}</h3>
+        </div>
+        <button class="remove-btn" data-id="${book.id}">Remove</button>
+       </div>
+  
+      `;
+    });
+    this.content.innerHTML = placeholder;
+  }
+
+  fetchBooks() {
+    this.renderBooks();
+  }
+}
+// eslint-disable-next-line no-unused-vars
```

Comment:
Kindly avoid disabling linter checks inline. They represent coding standards that you share with your team.